### ISSUE TITLE
dacauthority hard coded on initial set up

### ISSUE THREAD

**Issue by lukestokes** [eosdac/daccustodian#40]
I'm working to get an example DAC up and running on Jungle and I noticed I can't call the updateconfig method because it uses the defaults which are set as dacauthority. I talked it through with michael and this seems like the best place to make an improvement: https://github.com/eosdac/daccustodian/blob/master/config.cpp#L4

Something like:

```
if (...config has been updated, config table isn't empty, etc...) {
require_auth(configs().authaccount);
} else {
require_auth(_self);
}
```

As it is now, I have to manually adjust the hpp file to change the default to my own dac authority account in order to update the config.

**Comment by dallasjohnson** [comment]
Good thoughts. It's a bit of a chicken and egg scenario. 😃. I can look into it.

**Comment by dallasjohnson** [comment]
Fixed in #42
